msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -929,6 +931,8 @@ public static INativeAuthPublicClientApplication createNativeAuthPublicClientApp
      * @param authority   The default authority to be used for the authority. If this is null, the default authority will be used.
      * @param redirectUri The redirect URI of the application.
      * @return An instance of INativeAuthPublicClientApplication.
+     *
+     * @deprecated This method is deprecated because optional parameter CAPABILITIES  cannot be included. Use createNativeAuthPublicClientApplication(Context, NativeAuthPublicClientApplicationConfiguration) instead.
```

Comment:
```suggestion
     * @deprecated This method is deprecated. Use createNativeAuthPublicClientApplication(Context, NativeAuthPublicClientApplicationConfiguration) instead.
```
No need to specify the reason

Reply:
Done.